config/swagger.yaml

```diff
@@ -2245,7 +2247,7 @@ definitions:
     type: object
     properties:
       oracle_pubkey:
-        type: string
+        $ref: '#/definitions/EncodedHashOrName'
```

Comment:
The [oracle consensus doc describes this field as "The oracle (address)"](https://github.com/aeternity/protocol/blob/0918c2a3262bb6e39e9eb589da380b248d006f3a/oracles/oracle_transactions.md#oracle-query-transaction): that seems clashing with name?

Reply:
so, should we adjust spec, this is basically the feature this PR implements